src/dapp/libexec/dapp/dapp-remappings

```diff
@@ -17,6 +17,11 @@ function findRemappings(prefix) {
   ls(`${prefix}/${process.env.DAPP_LIB}`).forEach(name => {
     var lib = `${prefix}/${process.env.DAPP_LIB}`
     var src = `${lib}/${name}/${process.env.DAPP_SRC}`.replace(/^.\//, "")
+    
+    // If the file has an extension, ignore.
+    if (name.includes(".")) {
```

Comment:
should we maybe early return if it's not a directory instead?

Reply:
yeah probably more reliable will fix

Reply:
done!